test/src/e2e_vm_tests/test_programs/should_pass/language/supertraits_with_trait_methods/src/main.sw

```diff
@@ -0,0 +1,56 @@
+script;
+
+use std::assert::assert;
+
+trait MyAdd {
+	fn my_add(self, other: Self) -> Self;
+}
+
+trait MyMul {
+	fn my_mul(self, other: Self) -> Self;
+}
+
+trait MyMath: MyAdd + MyMul {
+
+} {
+	fn my_double(self) -> Self {
```

Comment:
and these 2 functions

Reply:
they are single-tab indents -- just tested and at the latest version of forc-fmt seems to replace tabs with four spaces.. 🤔

Reply:
Ack.